tests/__snapshots__/package/with-vocab/dist/index.ts.snap

```diff
@@ -3,8 +3,8 @@
 const jsxRuntime = require("react/jsx-runtime");
 const React = require("react");
 const client$1 = require("react-dom/client");
-const _vocab_index_cjs = require("./.vocab/index.cjs");
-const client_vocab_index_cjs = require("./client.vocab/index.cjs");
+const _vocab_index_ts = require("./.vocab/index.cjs");
```

Comment:
Unsure what caused this change. It also reads a bit weird, but it's bundled code so it doesn't really matter.

Reply:
I know why: [`3f52861` (#167)](https://github.com/seek-oss/crackle/pull/167/commits/3f52861e7f0c7f75b35483e3db5f26f85a109d6c)